modules/installation-overview.adoc

```diff
@@ -6,6 +6,8 @@
 :_content-type: CONCEPT
 [id="installation-overview_{context}"]
 
```

Comment:
Remove line 8 so there is no space between the id and the heading (https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata)